edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -717,8 +717,13 @@ impl ModuleRuntime for DockerModuleRuntime {
                             .flat_map(|container| {
                                 DockerConfig::new(
                                     container.image().to_string(),
-                                    ContainerCreateBody::new()
-                                        .with_labels(container.labels().clone()),
+                                    ContainerCreateBody::new().with_labels(
+                                        container
+                                            .labels()
+                                            .iter()
+                                            .map(|(k, v)| (k.to_string(), v.to_string()))
+                                            .collect::<BTreeMap<_, _>>(),
```

Comment:
Don't need the turbofish. `collect()`'s return type will be inferred from the `with_labels` signature.

Reply:
Fixed.